build/python/build/netcdfcxx.py

```diff
@@ -0,0 +1,6 @@
+from build.autotools import AutotoolsProject
+
+class NetcdfCxxProject(AutotoolsProject):
+    def get_make_args(self, toolchain):
+        # HELP TODO: Temp workaround. Will ANDROID_NDK alwats be set in env if in the default location?
```

Comment:
No, but this whole thing looks completely wrong anyway. What are you trying to do here?

Reply:
I found it is not needed. AutotoolsProject is good enough. Will be removed.